myapp/templates/index.html

```diff
@@ -33,18 +33,7 @@
 </div>
 
 {% load myapp_tags %}
-<div class=" row  row-m-t ">
-    <div class="col-6 text-left custom-control custom-switch">
-  <input type="checkbox" class="custom-control-input" id="customSwitch1"
-         onclick='handleClick();' >
-  <label class="custom-control-label" for="customSwitch1">Auto Update</label>
-        </div>
-    <div class="col-6 float-right text-right">
-          <p  >Last Update <span id="timer"></span></p>
-
-    </div>
-</div>
-
+{% include "subtemplate/Auto_update_control.html" %}
```

Comment:
It is good that you moved this to a seprate file and you are reusing the code in both index and watchlist